bucoffea/limit/compare_templates.py

```diff
@@ -112,11 +155,18 @@ def main():
 
         # Aesthetics
         ax.set_title(key)
-        rax.set_ylim(0,2)
-        rax.set_ylabel("Recoil (GeV)")
+        rax.set_ylim(*map(float, args.rlim.split(",")))
+        rax.set_xlabel("Recoil (GeV)")
+        rax.set_ylabel("Ratio")
         ax.set_ylabel("Events / bin")
-        fig.savefig(pjoin(outdir, f"{key}.png"))
+        ax.set_yscale("log")
+        ax.set_ylim(
+            0.5*min(h2[key].values[h2[key].values>0]),
```

Comment:
Hey Andreas,
Thanks, the plots look nice! I just encountered a quick issue regarding the line here. Since sometimes the keys are not relevant for the region we're considering, values end up to be a set of 0s. Then code crashes here because of >0 condition, can we make this >= 0 instead? (It worked for me, with a bunch of warnings when it encounters a set of 0s)

Or maybe we can find a way to avoid unnecessary looping as well, thanks!

Reply:
Thanks for checking! I now just put a try/except block around it. Setting y limits to 0 for a log plot just hurts my brain too much, even if its a useless plot :D better not do anything in that case

Reply:
You're right :D This version worked for me, thanks for the fix!